compiler/src/iree/compiler/Codegen/LLVMCPU/Utils.cpp

```diff
@@ -12,60 +12,89 @@
 
 #define DEBUG_TYPE "iree-llvmcpu-utils"
 
+static const char kMaxStackAllocationSizeAttrName[] =
+    "max_stack_allocation_size";
+static const char kNativeVectorSizeAttrName[] = "native_vector_size";
```

Comment:
nit / optional: I think it's more idiomatic to do
```suggestion
constexpr char kMaxStackAllocationSizeAttrName[] =
    "max_stack_allocation_size";
constexpr char kNativeVectorSizeAttrName[] = "native_vector_size";
```
`constexpr` implies `static` in the global namespace